test/cufinufft3d1_test.cu

```diff
@@ -101,6 +101,8 @@ int main(int argc, char* argv[])
 	ier=cufinufft_default_opts(type, dim, &dplan.opts);
 	dplan.opts.gpu_method=method;
 	dplan.opts.gpu_kerevalmeth=1;
+
+	/* if not set, some hard coded values for the bin size are used */
```

Comment:
Won't the values go away for method 1 and 2 as well?

Reply:
Yes, I am thinking of having them there as an example to show that users can set the options to values they'd like to try.

Reply:
but if you think this is not necessary and somehow confusing, I can remove them too.

Reply:
I think it might be confusing. Most readers (including myself) will read this as “you have to do this” as opposed to “you can do this”. Again, I expect a very small portion of users will actually want to fiddle with this.

Reply:
I see. Thanks for the feedback!  I removed them in 3988f01.

Reply:
Great!